### PR TITLE
Update styling on the scatter tool

### PR DESCRIPTION
**Pull request by beets**
* Migrate from jquery 1.11 to the base version from base.html (3.5.1)
* Remove jquery.ui
* Cleaned up the plot selection form
* Cleaned up dialog
* Added welcome message

image

### CODE REVIEW

File: server/templates/tools/scatter.html [hunk 1]
```diff
@@ -117,8 +130,54 @@
...
-{% endblock %}
\ No newline at end of file
+
+<div id='screen'>
+  <div id="spinner"></div>
+</div>
+{% endblock %}
+
+{% block footer %}
+<script src="https://code.jquery.com/ui/1.11.1/jquery-ui.min.js"></script>
```
Comment (shifucun) [inline server/templates/tools/scatter.html:176]: do you want to remove this from the PR description?

**Comment by shifucun** [comment]
Nice to fix the Codacy error as well. Wonder if there is a way to run the Codacy lint locally

File: server/templates/tools/scatter.html (hunk 1 above)
Comment (beets) [reply server/templates/tools/scatter.html]: Removed from code - it's not required anymore, required elements are taken from bootstrap, which is included in base.html

**Comment by beets** [comment]
Updated to also fix codacy warning, removed the Filter button, updated form validation display and some bugs as I was testing. PTAL

**Pull request merged**